components/Admin/OrderContent.js

```diff
@@ -22,7 +22,13 @@ export const OrderContent = ({
   isLoading,
   handleDeleteOrder,
 }) => {
-  const [orderDate, orderTime] = getSimpleDate(order.date);
+  const { formattedDateTime: orderDateTime } = getReadableDate(order.date);
+  const reason =
+    order.reason === 'item_restock'
+      ? 'Restock'
+      : order.reason === 'unserviceable'
+      ? 'Unserviceable'
+      : 'Others';
```

Comment:
instead of using ternary here, perhaps we can instead 

add this into globals.js
```js
export const ORDER_REASONS = {
  item_restock: 'Restock',
  unserviceable: 'Unserviceable',
  others: 'Others',
};
```

```suggestion
  const reason = ORDER_REASONS[order.reason];
```